jwst/refpix/tests/test_refpix.py

```diff
@@ -291,15 +295,15 @@ def test_above_sigma():
 
 
 def test_nan_refpix():
-    '''Verify that the reference pixels flagged DO_NOT_USE are not used in the calculation'''
+    '''Verify that the reference pixels flagged DO_NOT_USE are not used in the calculation
 
-    # Test that flagging a reference pixel with DO_NOT_USE does not use the pixel in the
-    # average. Set the pixel to NaN, which results in a NaN average value if used. If the test
-    # passes, then the NaN was correctly flagged and rejected from the average.
+    Test that flagging a reference pixel with DO_NOT_USE does not use the pixel in the
+    average. Set the pixel to NaN, which results in a NaN average value if used. If the test
+    passes, then the NaN was correctly flagged and rejected from the average.'''
 
     # create input data
     # create model of data with 0 value array
-    ngroups = 5
+    ngroups = 10
```

Comment:
Is there a reason the number of groups was increased?  I assume this will make the tests run slower?

Reply:
Thanks, I don't know why I did this... I went ahead and changed it back to 5.